### ISSUE TITLE
Howto Use DCC_SetGuildMemberNickname And DCC_AddGuildMemberRole??

### ISSUE THREAD

**Issue by kimpolartt** [maddinat0r/samp-discord-connector#178]
please help me ,, I don't really understand

**Comment by Sync-07** [comment]
yes, please answer this question, I'm now starting to think that **DCC_SetGuildMemberNickname** has a bug

**Comment by Courge-Adams** [comment]
new DCC_Guild:guild = DCC_FindGuildById("Your Server Id");
new DCC_User:user;
DCC_SetGuildMemberNickname(guild, user, "nickname");
